cmd/root.go

```diff
@@ -120,6 +120,8 @@ any client SSL certificates.`,
 		"Address on which to bind Cloud SQL instance listeners.")
 	cmd.PersistentFlags().IntVarP(&c.conf.Port, "port", "p", 0,
 		"Initial port to use for listeners. Subsequent listeners increment from this value.")
+	cmd.PersistentFlags().StringVarP(&c.conf.UnixSocket, "unix-socket", "u", "",
+		`Enables Unix sockets for all listeners using the provided directory.`)
```

Comment:
Should this default to `/cloudsql` if set with no argument?

Reply:
Since we require users to opt-in to Unix sockets, omitting the default is how we determine if the user did in fact set the flag.

Reply:
And even if we did set a default, we'd never use it given the opt-in nature of the flag.

Reply:
So I tried this out. If I supply a default and pass the flag without any value, e.g., `--unix-socket` without a directory, Cobra errors. So the default only applies if a user doesn't pass the flag. So I think we're OK to leave this as an empty string.